vgscollect/src/main/java/com/verygoodsecurity/vgscollect/widget/SSNEditText.kt

```diff
@@ -24,10 +24,11 @@ class SSNEditText @JvmOverloads constructor(
             0, 0
         ).apply {
             try {
-                val numberDivider:String? = getString(R.styleable.SSNEditText_numberDivider)
-                val outputNumberDivider:String? = getString(R.styleable.SSNEditText_outputNumberDivider)
+                val numberDivider: String = getString(R.styleable.SSNEditText_numberDivider) ?: "-"
```

Comment:
@Endorf I will suggest using `DIVIDER` constant from `SSNInputField`, because if constant will change it will be changed here to.